### DISCUSSION TITLE
Add gems ONLY as ingame items, for crossplay too?

### DISCUSSION THREAD

**Discussion by KyleLogan** [gameserverapp/Platform#779] [Suggestions]
Hi Max, loving GSA and your work, i wanted to ask if theres plan to add in future the Gems items also for crossplay servers cause they're unavailable for them. Keep the good work nad have a good day!

**Comment by mvaessen** [comment]
hi, thanks for your kind words!

Unfortunately were the gems removed due to CurseForge forcing us to do so. It's unlikely that they will allow anything like that for  ARK:SA console mods in the future.
